### ISSUE TITLE
Gradle 5 support needs to be added

### ISSUE THREAD

**Issue by tufandevrim** [NimbleDroid/gradle-profiler#17]
Gradle 5 support needs to be added

**Comment by dreuxl** [comment]
We have this kind of console output logs on our pipeline 

```
18:27:33   PROFILING: Profiling is currently running
18:28:03   PROFILING: Profiling is currently running
18:28:33   PROFILING: Profiling is currently running
18:29:03   PROFILING: Profiling is currently running
18:29:33   PROFILING: Profiling is currently running
18:30:03   PROFILING: Profiling is currently running
18:30:34   PROFILING: Profiling is currently running
18:31:04   PROFILING: Profiling is currently running
18:31:34   PROFILING: Profiling is currently running
18:32:04   NimbleDroid Gradle Plugin timed out.
18:32:04   You can contact support@nimbledroid.com if you need assistance.
18:32:04   :app:ndProfile (Thread[Task worker for ':',5,main]) completed. Took 1 hrs 30 mins 39.656 secs.
18:32:04  
18:32:04   Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
18:32:04   Use '--warning-mode all' to show the individual deprecation warnings.
18:32:04   See https://docs.gradle.org/4.10.1/userguide/command_line_interface.html#sec:command_line_warnings
```

**Comment by checkthebounds** [comment]
Addressed by #18
